monix-catnap/shared/src/main/scala/monix/catnap/ConcurrentQueue.scala

```diff
@@ -277,6 +277,15 @@ final class ConcurrentQueue[F[_], A] private (
     notifyProducers()
   }
 
+  /** Checks if the queue is empty.
+    *
+    * '''UNSAFE PROTOCOL:''' Concurrent shared state changes very frequently, therefore this function might yield nondeterministic results.
```

Comment:
I think it could use a new line. Also maybe similar warning to `tryPoll` would be more accurate, i.e. it implies an understanding of concurrency for given use case. I don't have any good idea about wording off the top of my head :D

Any ideas @alexandru ?

Reply:
ok, i wrote it in the manner you suggested, do you think it's correct more or less/should I add something more?